### DISCUSSION TITLE
AI運用在修圖上的技術

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS3073-C/Python-Community-GS3073-C#475] [科技新知]
這次應用程式的更新，是借助AI的技術去增強原有的功能，可見未來將會運用到更多層面而使生活更加便利。
https://digiphoto.techbang.com/posts/11799-ai-blessing-after-the-system-is-easier-adobe-lightroom-classic-version-114-is-a-retouching-demonstration-of-major-new-features

**Comment by lx3211** [comment]
直接強調修飾某部分的色彩
可以讓原本就很出色的照片，變得更令人驚艷耶

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by xCk27x** [comment]
沒想到Adobe也將AI技術整合到他們的產品了!

**Comment by williamyen0623** [comment]
現在什麼技術都能藉由AI達到優化原有功能甚至是開發新功能的目的，再次感嘆AI技術的強大！

**Comment by NcuJerryKao** [comment]
或許之後人們要PO照片上社群媒體前 就會靠AI來修圖

**Comment by Ben6uo** [comment]
可能之後交友綱站上的照片都不是真人

**Comment by 32844583** [comment]
如果搭配 Dell-E 運用在藝術作品上應該會備受欽佩吧 !

**Comment by limitx0** [comment]
看到遮色片可以用 AI 自動選取範圍真的是太棒了
省下超多時間
